libraries/AP_NavEKF2/AP_NavEKF2.cpp

```diff
@@ -1712,11 +1712,11 @@ void NavEKF2::writeDefaultAirSpeed(float airspeed)
  * timeStamp_ms : system time the measurement was taken, not the time it was received (mSec)
  * delay_ms   : average delay of external nav system measurements relative to inertial measurements
  */
-void NavEKF2::writeExtNavVelData(const Vector3f &vel, uint32_t timeStamp_ms, uint16_t delay_ms)
+void NavEKF2::writeExtNavVelData(const Vector3f &vel, float err, uint32_t timeStamp_ms, uint16_t delay_ms)
```

Comment:
a cosmetic thing but we should add a description of the new "err" parameters a few lines up..

Reply:
I miss this one, Thank you